tests/cephadm/test_osd_crush_device_class.py

```diff
@@ -0,0 +1,34 @@
+from ceph.waiter import WaitUntil
+from cli.cephadm.cephadm import CephAdm
+from cli.exceptions import ConfigError, OperationFailedError, OsdOperationError
+
+
+def run(ceph_cluster, **kwargs):
+    """Verify osd spec class crush_device_class"""
+    # Get config
+    config = kwargs.get("config")
+
+    # Check mandatory parameter crush_device_class
+    if not config.get("crush_device_class"):
+        raise ConfigError("Mandatory config 'crush_device_class' not provided")
+
+    # get cluster config
+    installer = installer = ceph_cluster.get_nodes(role="installer")[0]
```

Comment:
any reason why multiple assignment to same variable ? 
installer = installer = ceph_cluster.get_nodes(role="installer")[0]

Reply:
Thanks @tintumathew10 

Updated

Reply:
Thanks Mohit